### DISCUSSION TITLE
Welcome to numato-gpio Discussions!

### DISCUSSION THREAD

**Discussion by clssn** [clssn/numato-gpio#6] [General]
## 👋 Welcome!
With this Discussions board I'd like to learn about 

  * How you use your Numato GPIO boards
  * Which exact devices you use
  * Under which circumstances you're willing to help me test other boards than my own 32 Port Numato USB GPIO boards
  * Your general impression of this project
  * Features most interesting to you

This is the place to share your thoughts. Don't hesitate! Thanks :-)

**Comment by zachdurrence** [comment]
Hey, I have just started using this library, and it works great.

I did have an idea. On the discover function, could it be modified to use serial.tools.list_ports.comports().

Then you could check vendor id to make sure it is numato labs and potentially device id to get the device type.

This would make the function usable on windows and Linux.

Thanks!
